h2o-py/tests/testdir_algos/stackedensemble/pyunit_stackedensemble_distributions.py

```diff
@@ -0,0 +1,521 @@
+                                             metalearner_algorithm="gbm")
+    se_class_1.train(x_class, y_class, train)
+
+    assert se_class_1.metalearner().actual_params.get("distribution") == "multinomial", \
+        "Expected distribution {} but got {}".format("multinomial",
+                                                     se_class_1.metalearner().actual_params.get("distribution"))
+
+    se_class_2 = H2OStackedEnsembleEstimator(training_frame=train,
+                                             validation_frame=test,
+                                             base_models=[drf_class, nb_class, gbm_class],
+                                             metalearner_algorithm="gbm")
+    se_class_2.train(x_class, y_class, train)
+
+    assert se_class_2.metalearner().actual_params.get("distribution") == "multinomial", \
+        "Expected distribution {} but got {}".format("multinomial",
+                                                     se_class_2.metalearner().actual_params.get("distribution"))
+
+    se_class_3 = H2OStackedEnsembleEstimator(training_frame=train,
+                                             validation_frame=test,
+                                             base_models=[nb_class, gbm_class, drf_class])
+    se_class_3.train(x_class, y_class, train)
+
+    assert se_class_3.metalearner().actual_params.get("family") == "multinomial", \
+        "Expected family {} but got {}".format("multinomial",
+                                               se_class_3.metalearner().actual_params.get("family"))
+
+    se_class_4 = H2OStackedEnsembleEstimator(training_frame=train,
+                                             validation_frame=test,
+                                             base_models=[gbm_class, drf_class, nb_class])
+    se_class_4.train(x_class, y_class, train)
+
+    assert se_class_4.metalearner().actual_params.get("family") == "multinomial", \
+        "Expected family {} but got {}".format("multinomial",
+                                               se_class_4.metalearner().actual_params.get("family"))
+
+    se_class_5 = H2OStackedEnsembleEstimator(training_frame=train,
+                                             validation_frame=test,
+                                             base_models=[drf_class, nb_class, gbm_class])
+    se_class_5.train(x_class, y_class, train)
+
+    assert se_class_5.metalearner().actual_params.get("family") == "multinomial", \
+        "Expected family {} but got {}".format("multinomial",
+                                               se_class_5.metalearner().actual_params.get("family"))
+
+
+if __name__ == "__main__":
+    pyunit_utils.run_tests([
+        infer_distribution_test,
+        infer_family_test,
+        infer_mixed_family_and_dist_test,
+        metalearner_obeys_metalearner_params_test,
+        infer_uses_defaults_when_base_model_doesnt_support_distributions_test,
+        basic_inference_works_for_DRF_and_NB_test,
+    ])
```

Comment:
I understand it's faster to write Py tests for this kind of logic — I do the same —, and you have great tests with good coverage.
But let's try to have at least basic scenarios tested in Java as well, thx.